router/core/request_context_fields.go

```diff
@@ -32,6 +33,24 @@ const (
 
 // Helper functions to create zap fields for custom attributes.
 
+func NewExpressionLogField(val any, key string, defaultValue any) zap.Field {
```

Comment:
Does exist other places where we have to use it?

Reply:
Just this